src/Experiment/enums.ts

```diff
@@ -11,7 +11,9 @@ export enum ASSIGNMENT_UNIT {
 
 export enum POST_EXPERIMENT_RULE {
   CONTINUE = 'continue',
+  // TO DO : Remove revert when frontend anad backend integrated with assign
```

Comment:
@tanmayjain3 typo in "frontend anad backend"

Reply:
Updated